app/models/list.rb

```diff
@@ -1,8 +1,17 @@
 class List < ActiveRecord::Base
-  has_many :items, -> { order(:created_at) }
+  has_many :items, -> { order(:updated_at) }
 
   validates :title, presence: true
   validates :title, uniqueness: true
 
   scope :including_items, -> { includes(:items).order(:created_at) }
+
+  def completed_items
+    items.select { |item| item.done? }
+  end
+
+  def incompleted_items
+    items.reject { |item| item.done? }
+  end
```

Comment:
@vlasiak, As long as this is a temporary solution and we clearly see those sets of items on UI, I let this go without a test for now.